Add `preserveDirectories` option (#353)
imagemin/imagemin: readme.md

Set the destination folder to where your files will be written. If no destination is specified, no files will be written.

##### preserveDirectories

Type: `boolean`

> If enabled the directory structure from the input is preserved.

Reviewer: ```suggestion
Preserve the source directory structure in the output.
```

Set the destination folder to where your files will be written. If no destination is specified, no files will be written.

##### preserveDirectories

Type: `boolean`\
Default: `false`

Preserve the source directory structure in the output.

##### basePath

Type: `string`\
Default: `''`

Ignore certain directories in the output.

```js
await imagemin(['assets/images/**/*.jpg'], {
	destination: 'dist',
	basePath: 'assets/images',
	preserveDirectories: true,
});
```

> - `assets/images/1.jpg` -> `dist/1.jpg`

Reviewer: ```suggestion
- `assets/images/1.jpg` → `dist/1.jpg`
```
